app/controllers/admin/users_controller.rb

```diff
@@ -40,6 +40,11 @@ def index
 
       @organization_limit = 3
       @organizations = Organization.order(name: :desc)
+
+      @users.each do |user|
```

Comment:
Do you think we should add some specs to make sure we're tagging the right users as banishable? It feels like there's a fair bit of logic in this and I don't think we have anything checking this currently 🤔

Reply:
Just a note that this is not working as in the manner that I think it was intended to work. `@banishable_user` will not get set on every user, but rather `@banishable_user` gets calculated based on the last user, and uses that value to determine what to set in every modal for every user. I'm working on fixing this at the moment and will "request changes" to avoid an accidental merge.